### PR TITLE
Add .NET Standard 2.0 to ASQ + NServiceBus Core 7

### PR DESCRIPTION
**Pull request by SeanFeldman**
All projects compile.

Storage library has introduced some API changes between `net452` and `netstandard2.0`. Those are marked with `//TODO:` comments for especial review.

- [x] Review TODOs
- [x] follow the steps in https://github.com/Particular/DotNetCoreLaunch/issues/7

### CODE REVIEW

File: src/AcceptanceTests/Sending/When_delaying_messages_natively.cs [hunk 1]
```diff
@@ -89,7 +89,7 @@ public class When_delaying_messages_natively : NServiceBusAcceptanceTest
             await delayedMessagesTable.ExecuteAsync(TableOperation.Insert(delayedMessageEntity)).ConfigureAwait(false);
         }
 
-        async Task<List<DynamicTableEntity>> GetDelayedMessageEntities()
+        async Task<IList<DynamicTableEntity>> GetDelayedMessageEntities()
```
Comment (danielmarbach) [inline src/AcceptanceTests/Sending/When_delaying_messages_natively.cs]: Not that it really matters but let's revert this change

File: src/Transport/DelayDelivery/NativeDelayDelivery.cs [hunk 2]
```diff
@@ -22,7 +22,10 @@ public NativeDelayDelivery(string connectionString, string delayedMessagesTableN
             delayedMessagesTable = CloudStorageAccount.Parse(connectionString).CreateCloudTableClient().GetTableReference(delayedMessagesTableName);
             // In the constructor to ensure we do not force the calling code to remember to invoke any initialization method.
             // Also, CreateIfNotExistsAsync() returns BEFORE the table is actually ready to be used, causing 404.
-            delayedMessagesTable.CreateIfNotExists();
+
+            // TODO: original code was calling delayedMessagesTable.CreateIfNotExists(); as it was not affected by the bug the async version had.
+            // In case async version still returns before table is created, add a small delay.
+            delayedMessagesTable.CreateIfNotExistsAsync().GetAwaiter().GetResult();
```
Comment (danielmarbach) [inline src/Transport/DelayDelivery/NativeDelayDelivery.cs]: Another option would be to fire off the task and assign it to a field. When one of the public methods of this class is called await the outcome of the task.

File: src/NServiceBus.AzureStorageQueues.TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj [hunk 3]
```diff
@@ -1,26 +1,29 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net452</TargetFramework>
+    <TargetFrameworks>net452;netcoreapp2.0</TargetFrameworks>
```
Comment (danielmarbach) [inline src/NServiceBus.AzureStorageQueues.TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj]: So the reasoning is that everything that is not prod lib that client will target against has no benefit of being targetted against netstandard?

File: src/AcceptanceTests/Sending/When_delaying_messages_natively.cs (hunk 1 above)
Comment (SeanFeldman) [inline src/AcceptanceTests/Sending/When_delaying_messages_natively.cs]: This was intentional. netstd has changed the returned result to be IList. To avoid unnecessary conditional compilation, method signature is changes. It's a test method @danielmarbach

File: src/Transport/DelayDelivery/NativeDelayDelivery.cs (hunk 2 above)
Comment (SeanFeldman) [inline src/Transport/DelayDelivery/NativeDelayDelivery.cs]: First we need to verify that the bug in the async version that was there before is resolved.
If that's the case, then yes, a task-based approach with await will be good.

File: src/NServiceBus.AzureStorageQueues.TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj (hunk 3 above)
Comment (SeanFeldman) [inline src/NServiceBus.AzureStorageQueues.TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj]: And also to be able to run as far as I understood.

File: src/AcceptanceTests/Sending/When_delaying_messages_natively.cs (hunk 1 above)
Comment (SeanFeldman) [inline src/AcceptanceTests/Sending/When_delaying_messages_natively.cs]: BTW, the sentence reads funny..
> Not that it really matters but let's revert this change

If it doesn't really matter, why the proposition? 😜

Comment (danielmarbach) [inline src/AcceptanceTests/Sending/When_delaying_messages_natively.cs]: From a perf standpoint, it does not matter since it is a test. Didn't know that the lib version changed this.

Comment (SeanFeldman) [inline src/AcceptanceTests/Sending/When_delaying_messages_natively.cs]: Sadly, they did.
Just like they removed some method overloads forcing you to seed `nulls` all over the place.

**Comment by SeanFeldman** [comment]
@Particular/azure-maintainers any ideas why this is failing on TC but not locally?

File: src/Tests/NServiceBus.AzureStorageQueues.Tests.csproj [hunk 4]
```diff
@@ -1,36 +1,39 @@
...
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" />
+    <PackageReference Include="RhinoMocks" Version="3.6.1" NoWarn="NU1701" />
     <PackageReference Include="Particular.CodeRules" Version="0.2.0" />
-    <PackageReference Include="ApiApprover" Version="6.1.0-beta2" />
-    <PackageReference Include="PublicApiGenerator" Version="6.1.0-beta2" />
     <ProjectReference Include="..\Transport\NServiceBus.AzureStorageQueues.csproj" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
```
Comment (SeanFeldman) [inline src/Tests/NServiceBus.AzureStorageQueues.Tests.csproj]: @core-v7 folks (@timbussmann @andreasohlund) in regards to my [question](https://github.com/Particular/DotNetCoreLaunch/issues/7#issuecomment-322947378) - the answer @andreasohlund provided [here](https://github.com/Particular/DotNetCoreLaunch/issues/7#issuecomment-322983255) is what I have in place, but it [fails](https://builds.particular.net/viewLog.html?buildId=227292&buildTypeId=NServiceBus_Transports_NServiceBusAzureStorageQueues_1Build) on the build server. Any ideas?

**Comment by SeanFeldman** [comment]
**Update**
It's compiling, **but** not ready for multiple targets and **doesn't** have all the [bells and whistles](https://github.com/Particular/DotNetCoreLaunch/issues/7) required for completeness. 
Also, TC is a bit [broken](https://particularsoftware.slack.com/archives/C03GKSAGX/p1503014813000004).

**Comment by andreasohlund** [comment]
>It's compiling, but not ready for multiple targets

What's not working when multi targeting?

**Comment by SeanFeldman** [comment]
@andreasohlund sorry, I didn't communicate the right info.
Compilation is working fine. It's the testing that was an issue. I have talked to @kbaley about future options and have a way forward for now. 
Small success celebration - ASQ is targeting full fw and .netstandard 🙂

**Comment by SeanFeldman** [comment]
@andreasohlund is [`Directory.Build.props`](https://github.com/Particular/NServiceBus/blob/f5ddc3145d37bc0e147bc1880b4266d5bb931456/src/Directory.Build.props) required?

**Comment by andreasohlund** [comment]
Strictly speaking no but it paves the way for us to update them more easily
and it avoid duplication so I'd recommend using it

**Comment by SeanFeldman** [comment]
In such case, the general list in
https://github.com/Particular/DotNetCoreLaunch/issues/7 should have the
entry.

**Comment by andreasohlund** [comment]
There is already an entry for adding the build props into /src ?

**Comment by SeanFeldman** [comment]
Damn, I copied the list and it was updated since then 😃 
Ok, will remove it and just follow the source.

File: src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj [hunk 5]
```diff
@@ -1,25 +1,25 @@
...
-    <PackageReference Include="System.Spatial" Version="5.8.2" />
-    <ProjectReference Include="..\Transport\NServiceBus.AzureStorageQueues.csproj" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" />
+
+    <!-- Force tests to run with the latest Storage library -->
+    <PackageReference Include="WindowsAzure.Storage" Version="8.*" />
+
+    <!-- Remove package is updated to work with .NET Standard 2.0 -->
+    <PackageReference Include="NServiceBus.Newtonsoft.Json" Version="1.2.0-alpha0017" NoWarn="NU1701" />
```
Comment (SimonCropp) [inline src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj]: this should now be `2.0.0-*`

Comment (SimonCropp) [reply src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj]: and it is now net standard compat

**Comment by SeanFeldman** [comment]
@Particular/azure-maintainers I've pushed more changes to complete the upgrade to NSB7.

Please review the changes.

File: src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj [hunk 6]
```diff
@@ -1,25 +1,24 @@
...
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.2" />
-    <PackageReference Include="NServiceBus.AcceptanceTesting" Version="6.3.3" />
-    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="6.3.3" />
-    <PackageReference Include="NServiceBus" Version="6.3.3" />
-    <PackageReference Include="NUnit" Version="3.7.1" />
-    <PackageReference Include="System.Spatial" Version="5.8.2" />
     <ProjectReference Include="..\Transport\NServiceBus.AzureStorageQueues.csproj" />
+    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="7.0.0-*" />
+    <PackageReference Include="NUnit" Version="3.*" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
```
Comment (SimonCropp) [inline src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj]: i have been going `15.*`

File: src/Directory.Build.props [hunk 7]
```diff
@@ -0,0 +1,35 @@
+<Project>
+
+  <ItemGroup>
+    <PackageReference Include="Terrajobst.PlatformCompat.Analyzers" Version="0.0.95-alpha" PrivateAssets="All" />
```
Comment (SimonCropp) [inline src/Directory.Build.props]: i dont think these add enough value

File: src/Directory.Build.targets [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+    <Authors>NServiceBus Ltd</Authors>
+    <Company>NServiceBus Ltd</Company>
+    <PackageLicenseUrl>http://particular.net/LicenseAgreement</PackageLicenseUrl>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <Copyright>Copyright 2010-$([System.DateTime]::UtcNow.ToString(yyyy)) NServiceBus. All rights reserved</Copyright>
+    <PackageTags>nservicebus servicebus msmq cqrs publish subscribe</PackageTags>
+    <PackageIconUrl>http://s3.amazonaws.com/nuget.images/NServiceBus_32.png</PackageIconUrl>
+    <PackageProjectUrl>https://docs.particular.net/nuget/$(PackageId)</PackageProjectUrl>
+    <PackageOutputPath>..\..\nugets</PackageOutputPath>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```
Comment (SimonCropp) [inline src/Directory.Build.targets]: u only want this in the projects that produce packages

File: src/NServiceBus.AzureStorageQueues.TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj [hunk 9]
```diff
@@ -1,26 +1,29 @@
...
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(SolutionDir)NServiceBus.snk</AssemblyOriginatorKeyFile>
-    <DebugType>full</DebugType>
-    <DebugSymbols>True</DebugSymbols>
-    <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```
Comment (SimonCropp) [inline src/NServiceBus.AzureStorageQueues.TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj]: put this in the directory settings

File: src/Transport/NServiceBus.AzureStorageQueues.csproj [hunk 10]
```diff
@@ -13,24 +13,25 @@
...
-    <PackageReference Include="WindowsAzure.Storage" Version="8.0.0" />
-    <PackageReference Include="NServiceBus" Version="6.0.0" />
-    <PackageReference Include="Obsolete.Fody" Version="4.2.2" />
-    <PackageReference Include="System.Spatial" Version="5.8.2" />
-    <PackageReference Include="Fody" Version="2.1.0" />
-    <PackageReference Include="Particular.CodeRules" Version="0.2.0" />
-    <PackageReference Include="NuGetPackager" Version="0.6.3" />
-    <PackageReference Include="GitVersionTask" Version="4.0.0-beta0012" />
+    <PackageReference Include="NServiceBus" Version="7.0.0-*" />
+    <PackageReference Include="NServiceBus.Newtonsoft.Json" Version="[2.0.0-*, 3.0.0)" />
```
Comment (SimonCropp) [inline src/Transport/NServiceBus.AzureStorageQueues.csproj]: @Particular/nservicebus-maintainers @Particular/azure-maintainers is there any concerns with having a dependency between `NServiceBus.AzureStorageQueues` and `NServiceBus.Newtonsoft.Json`?

File: src/Transport/NServiceBus.AzureStorageQueues.csproj [hunk 11]
```diff
@@ -13,24 +13,25 @@
...
     <PackageReference Include="Microsoft.Data.Services.Client" Version="5.8.2" />
-    <PackageReference Include="WindowsAzure.Storage" Version="8.0.0" />
-    <PackageReference Include="NServiceBus" Version="6.0.0" />
-    <PackageReference Include="Obsolete.Fody" Version="4.2.2" />
-    <PackageReference Include="System.Spatial" Version="5.8.2" />
-    <PackageReference Include="Fody" Version="2.1.0" />
-    <PackageReference Include="Particular.CodeRules" Version="0.2.0" />
-    <PackageReference Include="NuGetPackager" Version="0.6.3" />
-    <PackageReference Include="GitVersionTask" Version="4.0.0-beta0012" />
+    <PackageReference Include="NServiceBus" Version="7.0.0-*" />
```
Comment (SimonCropp) [inline src/Transport/NServiceBus.AzureStorageQueues.csproj]: this should be a range

File: src/Transport/NServiceBus.AzureStorageQueues.csproj [hunk 12]
```diff
@@ -13,24 +13,25 @@
...
+  <PropertyGroup>
+    <PackageId>NServiceBus.Azure.Transports.WindowsAzureStorageQueues</PackageId>
+    <Description>NServiceBus Transport for the Windows Azure Storage Queues</Description>
+  </PropertyGroup>
+
   <ItemGroup>
-    <None Include="..\..\packaging\nuget\nservicebus.azurestoragequeues.nuspec" Link="nservicebus.azurestoragequeues.nuspec" />
-    <PackageReference Include="Janitor.Fody" Version="1.4.1" />
+    <!-- lock WindowsAzure.Storage to version 8.0.0 -->
+    <PackageReference Include="WindowsAzure.Storage" Version="8.0.0" />
```
Comment (SimonCropp) [inline src/Transport/NServiceBus.AzureStorageQueues.csproj]: why isnt this a range?

File: src/Transport/NServiceBus.AzureStorageQueues.csproj [hunk 13]
```diff
@@ -13,24 +13,25 @@
...
     <PackageReference Include="Microsoft.Azure.KeyVault.Core" Version="2.0.4" />
```
Comment (SimonCropp) [inline src/Transport/NServiceBus.AzureStorageQueues.csproj]: i think Microsoft.Azure.KeyVault.Core, Microsoft.Data.OData and System.Linq.Queryable are all redundant

File: src/Transport/NServiceBus.AzureStorageQueues.csproj [hunk 14]
```diff
@@ -13,24 +13,25 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <OutputPath>..\..\binaries\</OutputPath>
```
Comment (SimonCropp) [inline src/Transport/NServiceBus.AzureStorageQueues.csproj]: this is redundant

File: src/Directory.Build.props (hunk 7 above)
Comment (SeanFeldman) [reply src/Directory.Build.props]: This is a part of the template promoted [here](https://github.com/Particular/DotNetCoreLaunch/issues/7)

File: src/Directory.Build.targets (hunk 8 above)
Comment (SeanFeldman) [reply src/Directory.Build.targets]: Not following. We do produce a single package, the transport package.

Comment (andreasohlund) [reply src/Directory.Build.targets]: Yes but this file applies to all projects causing the test projects to output nugets as well

See https://github.com/Particular/NServiceBus.Gateway/blob/develop/src/NServiceBus.Gateway/NServiceBus.Gateway.csproj#L11

File: src/NServiceBus.AzureStorageQueues.TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj (hunk 9 above)
Comment (SeanFeldman) [reply src/NServiceBus.AzureStorageQueues.TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj]: > put this in the directory settings

Example please?

File: src/Transport/NServiceBus.AzureStorageQueues.csproj (hunk 12 above)
Comment (SeanFeldman) [reply src/Transport/NServiceBus.AzureStorageQueues.csproj]: Should be, will fix.

File: src/Transport/NServiceBus.AzureStorageQueues.csproj (hunk 13 above)
Comment (SeanFeldman) [reply src/Transport/NServiceBus.AzureStorageQueues.csproj]: Those are [dependencies](https://www.nuget.org/packages/WindowsAzure.Storage/8.0.0) for `WindowsAzure.Storage`. Can I drop those and NuGet will identify those?

File: src/Directory.Build.targets (hunk 8 above)
Comment (SeanFeldman) [reply src/Directory.Build.targets]: @andreasohlund so alternatively, I could move that instruction specifically into the product.csproj file. Would that work?

Comment (andreasohlund) [reply src/Directory.Build.targets]: That is the preferred way to do it

File: src/Directory.Build.props (hunk 7 above)
Comment (SeanFeldman) [reply src/Directory.Build.props]: It might be generated false alarms, but aside from that, does it hurt to have it?

Comment (SimonCropp) [reply src/Directory.Build.props]: well since we should be actually running our acc tests on the target platforms, any problematic APIs should be caught as part of that. so for me it is redundant

File: src/NServiceBus.AzureStorageQueues.TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj (hunk 9 above)
Comment (SimonCropp) [reply src/NServiceBus.AzureStorageQueues.TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj]: https://github.com/Particular/NServiceBus.Wcf/blob/develop/src/Directory.Build.props#L4

Comment (SeanFeldman) [reply src/NServiceBus.AzureStorageQueues.TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj]: Will move to `Directory.Build.props`

File: src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj [hunk 15]
```diff
@@ -1,25 +1,24 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net452</TargetFramework>
+    <TargetFrameworks>net452;netcoreapp2.0</TargetFrameworks>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <DebugSymbols>True</DebugSymbols>
     <WarningLevel>4</WarningLevel>
```
Comment (SimonCropp) [inline src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj]: `<WarningLevel>4</WarningLevel>` is the default and can be removed

File: src/NServiceBus.AzureStorageQueues.TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj [hunk 16]
```diff
@@ -1,26 +1,29 @@
...
-    <PackageReference Include="NServiceBus.Testing.Fakes.Sources" Version="6.3.3" />
-    <PackageReference Include="NServiceBus.TransportTests.Sources" Version="6.3.3" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" />
+    <PackageReference Include="NServiceBus.Testing.Fakes.Sources" Version="7.0.0-*" />
+    <PackageReference Include="NServiceBus.TransportTests.Sources" Version="7.0.0-*" />
     <PackageReference Include="Microsoft.Azure.KeyVault.Core" Version="2.0.4" />
-    <PackageReference Include="Microsoft.Data.Edm" Version="5.8.2" />
-    <PackageReference Include="Microsoft.Data.OData" Version="5.8.2" />
     <PackageReference Include="Microsoft.Data.Services.Client" Version="5.8.2" />
```
Comment (SimonCropp) [inline src/NServiceBus.AzureStorageQueues.TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj]: should be a range?

File: src/Tests/NServiceBus.AzureStorageQueues.Tests.csproj [hunk 17]
```diff
@@ -1,36 +1,43 @@
...
-    <PackageReference Include="WindowsAzure.Storage" Version="8.1.4" />
-    <PackageReference Include="System.Spatial" Version="5.8.2" />
+    <None Remove="APIApprovals.ApproveAzureStorageQueueTransport.approved.txt" />
+    <None Remove="APIApprovals.ApproveAzureStorageQueueTransport.received.txt" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="NUnit" Version="3.*" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
```
Comment (SimonCropp) [inline src/Tests/NServiceBus.AzureStorageQueues.Tests.csproj]: duplicate ref

File: src/Transport/NServiceBus.AzureStorageQueues.csproj (hunk 13 above)
Comment (SimonCropp) [reply src/Transport/NServiceBus.AzureStorageQueues.csproj]: > Can I drop those and NuGet will identify those? 

yep. and same for System.Linq.Queryable

File: src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj [hunk 18]
```diff
@@ -1,25 +1,24 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net452</TargetFramework>
+    <TargetFrameworks>net452;netcoreapp2.0</TargetFrameworks>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <DebugSymbols>True</DebugSymbols>
```
Comment (SimonCropp) [inline src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj]: `<DebugSymbols>True</DebugSymbols>` why ?

File: src/Transport/NServiceBus.AzureStorageQueues.csproj (hunk 13 above)
Comment (SeanFeldman) [reply src/Transport/NServiceBus.AzureStorageQueues.csproj]: If I do so, get an error:
```error NU1701: Package 'System.Linq.Queryable 4.0.0' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v2.0'. This package may not be fully compatible with your project.```

File: src/Directory.Build.props (hunk 7 above)
Comment (SimonCropp) [reply src/Directory.Build.props]: and my reason for disliking them is the perf impact. https://developercommunity.visualstudio.com/content/problem/67259/roslyn-analyzers-are-being-slow-msbuild-using-lots.html  https://stackoverflow.com/questions/36461174/how-do-i-disable-all-roslyn-code-analyzers

it is a perf impact u pay on every keypress, and sometime even when idle. and the loss of RAM. if i were to chose a time to run this type of Analysis it would be in a unit test.

File: src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj (hunk 18 above)
Comment (SeanFeldman) [reply src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj]: Don't know. Ask @SimonCropp 😛 https://github.com/Particular/NServiceBus.AzureStorageQueues/blame/4ca310067c521619550de5622a3b8d4b9bf75f56/src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj

Comment (SimonCropp) [reply src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj]: i suspect that was a copy paste issue?

File: src/Directory.Build.props (hunk 7 above)
Comment (SeanFeldman) [reply src/Directory.Build.props]: 👍 I'll disable it for now.

File: src/Transport/NServiceBus.AzureStorageQueues.csproj (hunk 10 above)
Comment (SeanFeldman) [reply src/Transport/NServiceBus.AzureStorageQueues.csproj]: @SimonCropp let's chat about this one?

File: src/Transport/NServiceBus.AzureStorageQueues.csproj (hunk 11 above)
Comment (SeanFeldman) [reply src/Transport/NServiceBus.AzureStorageQueues.csproj]: Can't be a range. We want to test against 7.0.0 and not the latest minor/hotfix
